docs/scripting/syntax/return_object

```diff
@@ -0,0 +1,40 @@
+---
+title: Return Object
+---
+
+Built in function, and many user owned scripts return an object that has special formatting within the terminal. These special objects are often refered to as a ((%LSuccess%)) or ((%DFailure%)) object.
```

Comment:
```suggestion
An object returned to the CLI from a script may be automatically formatted, depending on its contents. These types of objects are often referred to as ((%LSuccess%)) or ((%DFailure%)) objects.
```

I think this wording might be clearer for someone looking for a brief explanation for "what is this".